vite-project/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
You don't really need Vite for the example repo. It's good practice but implementing it is actually a long and painful process (I tried it). So you could remove vite configuration to simplify this PR. If you want to do this it's worth doing it in a new branch in case it goes wrong :)